doc/ref/stdlib.html

```diff
@@ -1037,25 +1223,24 @@ <h3 id="manifestation">Manifestation</h3>
 <div class="hgroup">
   <div class="hgroup-inline">
     <div class="panel">
-      <h4 id="manifestIni">std.manifestIni(ini)</h4>
+      <h4 id="manifestIni">
+        std.manifestIni(ini)
+      </h4>
     </div>
     <div style="clear: both"></div>
   </div>
 </div>
-
 <div class="hgroup">
   <div class="hgroup-inline">
     <div class="panel">
       <p>
-        Convert the given structure to a string in <a href="http://en.wikipedia.org/wiki/INI_file">INI format</a>. This
-        allows using Jsonnet's
-        object model to build a configuration to be consumed by an application expecting an INI
-        file. The data is in the form of a set of sections, each containing a key/value mapping.
-        These examples should make it clear:
-      </p>
-
-
-      <pre>{
+    Convert the given structure to a string in <a href="http://en.wikipedia.org/wiki/INI_file">INI format</a>. This
+    allows using Jsonnet's
+    object model to build a configuration to be consumed by an application expecting an INI
+    file. The data is in the form of a set of sections, each containing a key/value mapping.
+    These examples should make it clear:
+</p>
```

Comment:
It's suddenly got broken indentation here, but was working fine further up the file.  Intended?

Reply:
The whole description was marked as verbatim html, so that <pre> wouldn't get messed up. This was intentional, but I've added a proper support for pre now.